cves/CVE-2015-0220.yml

```diff
@@ -224,37 +268,41 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+      Input should never be naitively trusted, and in this case was never
+      properly being verified
   security_by_obscurity:
-    applies:
+    applies: false
     note:
   serial_killer:
-    applies:
+    applies: false
     note:
   environment_variables:
-    applies:
+    applies: false
     note:
   secure_by_default:
-    applies:
+    applies: false
     note:
   yagni:
-    applies:
-    note:
+    applies: true
+    note: |
+      It appears the developers here only implemented .split when it was
```

Comment:
This doesn't match the previous style you had been writing with whereas you were using high level language to analyze the CWE. You refer to .split like something everyone should know, I would recommend using the same high level language again to keep with the current style.